From 8c92fcb13a4979232787f0476bf7469ccbb03617 Mon Sep 17 00:00:00 2001
From: Jonathan Wakely <jwakely@redhat.com>
Date: Mon, 27 Dec 2021 16:20:24 +0000
Subject: [PATCH] aarch64: Fix mismatched extern "C" block [PR100985]

gcc/ChangeLog:

	PR target/100985
	* config/aarch64/arm_acle.h: Remove unclosed extern "C" block.
---
 gcc/config/aarch64/arm_acle.h | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/gcc/config/aarch64/arm_acle.h b/gcc/config/aarch64/arm_acle.h
index 56147352c23..8c622a3985b 100644
--- a/gcc/config/aarch64/arm_acle.h
+++ b/gcc/config/aarch64/arm_acle.h
@@ -98,10 +98,6 @@ __rint64x (double __a)
 
 #pragma GCC target ("+nothing+crc")
 
-#ifdef __cplusplus
-extern "C" {
-#endif
-
 __extension__ static __inline uint32_t __attribute__ ((__always_inline__))
 __crc32b (uint32_t __a, uint8_t __b)
 {
-- 
2.31.1


